### ISSUE TITLE
No swift support

### ISSUE THREAD

**Issue by Pyrolr** [Tanchey/RainbowBrackets#1]
Not working for swift 
Working good for the objective- C, but it didn't changed anything for the swift

If you have any idea about how swift support can be added please let me know i may try from my side 

Thanks

**Comment by Tanchey** [comment]
Thanks for your interest in my tiny project.
Swift support appeared to be more complicated than I thought. And I'm working on it.

I've looked into XCode headers: https://github.com/zolomatok/Xcode6-RuntimeHeaders. I believe `IDESourceLanguageServiceSwift` may help, but I did not try it yet. (I'll try just as soon as I find time to do it)

So if you succeed first, please let me know or better send a pull request.

**Comment by Pyrolr** [comment]
Yes, I like your concept and i thought it will more useful if it supports swift also 
I will check if i may find something useful i will let you know

**Comment by mbrueck** [comment]
@Tanchey after 8 years, did you find a way to solve that issue? I ask for a friend :D
